openwpm/DataAggregator/base_aggregator.py

```diff
@@ -193,10 +193,10 @@ class BaseAggregator:
 
     Parameters
     ----------
-    manager_params : dict
+    manager_params : ManagerParamsInternal
         TaskManager configuration parameters
-    browser_params : list of dict
-        List of browser configuration dictionaries"""
+    browser_params : list of BrowserParamsInternal
+        List of browser configuration class<BrowserParams>"""
```

Comment:
Could you please also add the type annotations for the actual `__init__` function